opentelemetry/proto/trace/v1/trace.proto

```diff
@@ -244,16 +245,42 @@ message Span {
   }
 
   // The included links.
-  Links links = 12;
+  Links links = 13;
 
   // An optional final status for this span. Semantically when Status
   // wasn't set it is means span ended without errors and assume
   // Status.Ok (code = 0).
-  Status status = 13;
+  Status status = 14;
 
   // An optional number of child spans that were generated while this span
   // was active. If set, allows an implementation to detect missing child spans.
-  google.protobuf.UInt32Value child_span_count = 14;
+  google.protobuf.UInt32Value child_span_count = 15;
+}
+
+// AttributeKeyValue is a key-value pair that is used to store Span attributes, Link
+// attributes, etc.
+message AttributeKeyValue {
+  // ValueType is the enumeration of possible types that value can have.
```

Comment:
I like that STRING is zero. Perhaps mention that if omitted - `string_value` will be used

Reply:
That's the default behavior of protobufs. It is not possible to distinguish omitted from 0 value on the receiving side and the sender automatically omits zero values when encoding, so you don't really need to do anything special about this, it will just work automatically like that (at least in languages which auto-initialize structs to zero values, like Go).
Do you think it is worth a comment?